runtime/directives/sky-app-link-external.directive.ts

```diff
@@ -0,0 +1,36 @@
+import { Directive, Input, HostListener, OnInit } from '@angular/core';
+import { SkyAppWindowRef } from '../window-ref';
+import { SkyAppConfig } from '@blackbaud/skyux-builder/runtime';
+import { HttpParams } from '@angular/common/http';
+
+@Directive({
+  selector: '[skyAppLinkExternal]'
+})
+export class SkyAppLinkExternalDirective implements OnInit {
+  @Input() private skyAppLinkExternal: string;
```

Comment:
This should be refactored to have a `get`ter and a `set`ter so that the `externalUrl` property can be updated whenever the value changes.  Right now you're only setting the URL when the directive is initialized.